js-tests/test-utils.js

```diff
@@ -18,6 +18,16 @@ export const mockReportFields = [
   { field_name: 'field_3', field_label: 'Field 3' }
 ];
 
+/**
+ * Function to build response from the data service's `fetchSecurityConfig` method.
+ * @param {Boolean} hasReportsRights - true to indicate the user has reports rights
+ */
+export const buildMockSecurityConfig = function(hasReportsRights=false) {
```

Comment:
Purely pedantic, but this might be better named as `mockSecurityConfig`, htough honestly I don't mind either way.